Sources/TuistKit/Commands/Dependencies/DependenciesUpdateCommand.swift

```diff
@@ -0,0 +1,22 @@
+import ArgumentParser
+import Foundation
+import TSCBasic
+
+/// A command to update project's dependencies.
+struct DependenciesUpdateCommand: ParsableCommand {
+    static var configuration: CommandConfiguration {
+        CommandConfiguration(commandName: "update",
+                             abstract: "Updates project's dependencies.")
```

Comment:
```suggestion
                             abstract: "Updates project's dependencies defined in Dependencies.swift.")
```

Reply:
Similar comment here in regards to addint the article to `project`.